@here/harp-mapview/lib/geometry/TileGeometryLoader.ts

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { DecodedTile } from "@here/harp-datasource-protocol";
+import { PerformanceTimer } from "@here/harp-utils";
+
+import { PerformanceStatistics } from "../Statistics";
+import { Tile } from "../Tile";
+import { TileGeometryCreator } from "./TileGeometryCreator";
+
+/**
+ * Loads the geometry for its [[Tile]]. Derived classes allow for different loading strategies.
+ */
+export interface TileGeometryLoader {
+    /**
+     * The [[Tile]] this `TileGeometryLoader` is managing.
```

Comment:
[[TileGeometryLoader]]

Reply:
I am pretty sure that within the class itself, its is `TileGeometryLoader`. Otherwise, we link onto ourselves.